### PR TITLE
Migrate changeCSS() to CSS variable in utils/utils.jsx, ln. 640

### PR DESCRIPTION
**Pull request by d-schaffer**
#### Ticket Link
Fixes https://github.com/mattermost/mattermost-server/issues/15896

#### Related Pull Requests

- Has server changes (please link here)
- Has redux changes (please link here)
- Has mobile changes (please link here)

### CODE REVIEW

**Comment by d-schaffer** [comment]
check-cla

File: sass/layout/_sidebar-menu.scss [hunk 1]
```diff
@@ -107,3 +107,10 @@
         }
     }
 }
+.app__body {
+    .modal-tabs {
+        .nav-tabs > li {
+            background-color: var(--center-channel-bg);
+        }
+    }
+}
```
Comment (nevyangelova) [inline sass/layout/_sidebar-menu.scss:116]: nit: new line

**Review by nevyangelova — APPROVED** [review]
Thanks! LGTM minus the little nit comment :)

**Review by asaadmahmood — APPROVED** [review]

File: utils/utils.jsx [hunk 2]
```diff
@@ -634,7 +634,7 @@ export function applyTheme(theme) {
         changeCss('.app__body .shortcut-key, .app__body .post-list__new-messages-below', 'color:' + theme.centerChannelBg);
         changeCss('.app__body .emoji-picker, .app__body .emoji-picker__search', 'background:' + theme.centerChannelBg);
         changeCss('.app__body .nav-tabs, .app__body .nav-tabs > li.active > a', 'background:' + theme.centerChannelBg);
-        changeCss('.app__body .modal-tabs .nav-tabs > li', `background:${theme.centerChannelBg}`);
+        changeCss('.app__body .modal-tabs .nav-tabs > li', 'background:' + theme.centerChannelBg);
```
Comment (nevyangelova) [inline utils/utils.jsx:637]: @d-schaffer can you remove this line? We are replacing the function call with the CSS variables.

**Review by nevyangelova — CHANGES_REQUESTED** [review]
Can you also change the target branch `campaign/applytheme_center-channel-bg`

File: utils/utils.jsx (hunk 2 above)
Comment (hanzei) [reply utils/utils.jsx:637]: @d-schaffer do you have questions about this feedback?

**Comment by jfrerich** [comment]
Hi @d-schaffer! Do you need any further clarification from our team?  It would be great to get your PR merged.

File: sass/layout/_sidebar-menu.scss (hunk 1 above)
Comment (deanwhillier) [inline sass/layout/_sidebar-menu.scss:116]:
```suggestion
}

```

File: utils/utils.jsx (hunk 2 above)
Comment (deanwhillier) [inline utils/utils.jsx:637]:
```suggestion
```

**Review by deanwhillier — APPROVED** [review]
Using suggestions to complete this PR, approved once they are applied.

**Comment by deanwhillier** [comment]
We need this to be merged into the campaign branch and not directly into master, so I pulled `https://github.com/d-schaffer/mattermost-webapp.git` directly into the branch and am closing this PR.

Thanks again for your help on this @d-schaffer!

**Comment by jasonblais** [comment]
@deanwhillier heads up that closing a PR is an indication that the changes were invalid and actually don't get counted as contributions (nor towards swag). We recommend instead of manually pulling the changes to work with the contributor to rebase changes to a different branch as appropriate, in order for their changes to still be counted as contributions.

**Pull request closed without merging**